cdk/kittyhawk/src/application.ts

```diff
@@ -1,17 +1,73 @@
 import { Construct } from 'constructs';
 import { Certificate } from './certificate';
 import { Deployment, DeploymentProps } from './deployment';
-import { Ingress, IngressProps } from './ingress';
-import { Service, ServiceProps } from './service';
+import { HostRules, Ingress, IngressProps } from './ingress';
+import { Service } from './service';
 
 /**
  * Warning: Before editing any interfaces, make sure that none of the interfaces will have
  * property names that conflict with each other. Typescript may not throw an error and it
  * could cause problems.
  */
-export interface ApplicationProps extends IngressProps, DeploymentProps,
-  ServiceProps { }
+export interface ApplicationProps {
```

Comment:
Someone please check the contents of this file to make sure the changes I made in the past few PRs are reasonable ty